examples/fields/DateSheet.spec.ts

```diff
@@ -0,0 +1,176 @@
+import * as _ from 'lodash'
```

Comment:
These are really tests for SmartDateField, not DateSheet. I understand why you need to instantiate a Sheet to run these tests, but the file name will confuse people. Rename to SmartDateField.spec.ts.

Reply:
Done